### PR TITLE
Unlimited Jump!

### PR DESCRIPTION
**Pull request by xAstroBoy**
just adds unlimited jump!

### CODE REVIEW

File: lc-hax/Scripts/Commands/UnlimJumpToggleCommand.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+using Hax;
+
+[Command("unlimjump")]
+internal class UnlimJumpToggleCommand : ICommand {
```
Comment (winstxnhdw) [inline lc-hax/Scripts/Commands/UnlimJumpToggleCommand.cs:4]:
```suggestion
[Command("jump")]
class UnlimitedJumpCommand : ICommand {
```

**Review by winstxnhdw — CHANGES_REQUESTED** [review]

**Comment by xAstroBoy** [comment]
@winstxnhdw anything else?

**Comment by winstxnhdw** [comment]
update README

File: README.md [hunk 2]
```diff
@@ -139,6 +139,7 @@ The complete feature set includes the following.
 | Get coordinates               | `/xyz`                                           |
 | Toggle God mode               | `/god`                                           |
 | Toggle NoClip                 | `/noclip`                                        |
+| Toggle Unlimited Jump         | `/jump`                                          |
```
Comment (winstxnhdw) [inline README.md:142]:
```suggestion
| Toggle unlimited Jump         | `/jump`                                          |
```

**Review by winstxnhdw — CHANGES_REQUESTED** [review]

**Review by winstxnhdw — APPROVED** [review]

**Comment by winstxnhdw** [comment]
@xAstroBoy I forgot to ask but what’s the point of unlimited jump when we already have noclip?

**Comment by xAstroBoy** [comment]
> @xAstroBoy I forgot to ask but what’s the point of unlimited jump when we already have noclip?

it looks more natural, and can be a alternative of noclip because you can jump your way to obstacles instead of flying.

**Pull request merged**